documentation/final-deliverables/pdf-report.md

```diff
@@ -109,3 +109,11 @@ FR27. Implement “bonus features” (a.k.a features that are not required of th
 ## Technical Specifications
 
  (needs to be added)
+
+<br>
+
+## Limitations of MentorMatch
+
+* The way a mentor's availability is obatined is through a Google Calendar link field on the mentor form. This field is currently limited to only accepted Google Calendar links.
+
+* Mentees and mentors communicate via email messages. Due to this, if MentorMatch was further developed and internal communication between mentor and mentee were created, mentees would also need a MentorMatch account.
```

Comment:
Can you also add in how Mentors will have to manually update their calendar to reflect booked timeslots with Google Calendars? [Google Appointment Slots](https://support.google.com/calendar/answer/190998?co=GENIE.Platform%3DDesktop&hl=en) helps to remediate this, but since we don't have access to an organization or school Google Calendar, it's not something that we could test. However, organizations will likely be able to use Appointment Slots. 
At the time of writing this, I have class in 10 minutes so I'm not able to fully review this PR. I'll come back to this later today though.

Reply:
I wrote this in the final pdf. So I was not sure if we want to write it again in the slides. Should I still write it in the slides?

Reply:
Sorry, I read your comment wrong! Yes I will add this into the limitations!

Reply:
Added!